### PR TITLE
Upstream CentOS now uses local mirror from Fileserver

### PR DESCRIPTION
**Pull request by BaptisteGerondeau**

### CODE REVIEW

**Comment by rengolin** [comment]
We should also change (or remove) the kernel cmdline arguments in the Jenkins job.

**Comment by BaptisteGerondeau** [comment]
Done !

**Comment by rengolin** [comment]
Both upstream and ERP worked with the local mirror, we should change both.

File: files/preseeds/Centos.kickstart [hunk 1]
```diff
@@ -59,5 +59,5 @@ chmod 0600 /root/.ssh/authorized_keys
 #restorecon -R /root/.ssh/
 
 # Add apt-cacher-ng to conf
-echo "proxy=http://10.40.0.13:3142" >> /etc/yum.conf
+echo "proxy=http://10.40.0.13:3142" >> /etc/yum.conf
```
Comment (rengolin) [inline files/preseeds/Centos.kickstart]: unneeded whitespace change

**Comment by rengolin** [comment]
Hum, what I said doesn't make sense. The preseed needs to be different as it sets the URL of both repos by hand. I'll comment the proxy line.

**Pull request merged**